src/Factory/RepositoryProvider.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\Factory;
+
+use Closure;
+use Cycle\ORM\ORMInterface;
+use Cycle\ORM\RepositoryInterface;
+use Cycle\ORM\SchemaInterface;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\Support\ServiceProvider;
+
+final class RepositoryProvider extends ServiceProvider
```

Comment:
Need phpdoc here and, likely, addition to overall documentation.

Reply:
Shouldn't it be in the `Provider` folder?